build.psm1

```diff
@@ -545,7 +545,7 @@ Fix steps:
 
     if ($Environment.IsUbuntu -or $Environment.IsDebian)
     {
-        # psrp 1.3* now depends on libgssapi_krb5 for Kerberos support.
+        # libpsl-native.so v1.3 now depends on libgssapi_krb5 for Kerberos support.
```

Comment:
`libpsl-native.so` is the binary powershell builds, not related to PSRP.

Reply:
fixed